CHANGES.rst

```diff
@@ -8,6 +8,8 @@ Unreleased
 
 * :meth:`praw.models.Subreddits.search_by_topic` to search subreddits by topic.
   (see: https://www.reddit.com/dev/api/#GET_api_subreddits_by_topic).
+* :meth:`praw.models.LiveThread.__init__` to provide explicit interface for
```

Comment:
Can we add the interface to this feature at this time as I want to discourage directly creating instances of the classes. The way I think we talked about creating instance of live threads was via:

    reddit.live(thread_id)

Then the changelog can link to this method rather than an `__init__` method.

With this it would be great to add a unit test similar to `test_comment` for this feature to ensure the LiveThread is lazy: https://github.com/praw-dev/praw/blob/de1caa50deed07f19c2530f7894e575ec1b88ea9/tests/unit/test_reddit.py#L29